packages/kolibri-common/strings/bulkUserManagementStrings.js

```diff
@@ -0,0 +1,235 @@
+import { createTranslator } from 'kolibri/utils/i18n';
+
+export const bulkUserManagementStrings = createTranslator('BulkUserManagementStrings', {
+  // Searching and filtering
+  newUser: {
+    message: 'New user',
+    context: 'Button label that lets the user create a new user',
+  },
+  searchForAUser: {
+    message: 'Search for a user',
+    context: 'Placeholder text for user search input',
+  },
+  numUsersSelected: {
+    message: '{n, number} {n, plural, one {user} other {users}} selected',
+    context: 'A label showing the number of users selected',
+  },
+  numFilters: {
+    message: '{n, number} {n, plural, one {filter} other {filters}}',
+    context: 'A label showing the number of filters active',
+  },
+
+  // Dropdown options
+  viewNewUsers: {
+    message: 'View new users',
+    context: 'Label for dropdown item that links user to page of recently added users',
+  },
+  viewTrash: {
+    message: 'View trash',
+    context: 'Label for dropdown item that links user to trash page of soft-deleted users',
+  },
+
+  // Bulk actions
+  enrollToClass: {
+    message: 'Enroll to class',
+    context:
+      'Label for bulk-action button that will allow user to enroll selected learners to classes',
+  },
+  removeFromClass: {
+    message: 'Remove from class',
+    context:
+      'Label for bulk-action button that will allow user to remove selected learners from classes',
+  },
+  assignCoach: {
+    message: 'Assign coach',
+    context:
+      'Label for bulk-action button that will allow user to assign selected coaches to classes',
+  },
+  deleteSelection: {
+    message: 'Delete selection',
+    context: 'Label for bulk-action button that will allow user to delete selected users',
+  },
+
+  // Selection warnings
+  numUsersNotEnrolled: {
+    message:
+      '{num, number} {num, plural, one {user is} other {users are}} are not enrolled in any class',
+    context:
+      'A notice indicating the number of users that are selected which are not enrolled in a class',
+  },
+  numUsersCoaches: {
+    message: '{num, number} {num, plural, one {user is} other {users are}} are coaches',
+    context: 'A notice indicating the number of users the user selected that are coaches',
+  },
+
+  // Assign coaches to class
+  undoAssignCoachHeading: {
+    message:
+      '{num, number} {num, plural, one {coach has} other {coaches have}} been removed. Undo this?',
```

Comment:
I will need to double check what exactly the guidance is here for verbs and how much of the verb needs to be included, but I don't think that need to be blocking. Although I will set myself a reminder to see if I can find some documentation on this and finalize during string review